java-client/src/main/java/com/cumulocity/sdk/client/RestConnector.java

```diff
@@ -272,8 +282,14 @@ public <T extends ResourceRepresentation> T post(String path, MediaType mediaTyp
 
     @Override
     public <T extends ResourceRepresentationWithId> T post(String path, MediaType mediaType, T representation) throws SDKException {
-        Response response = httpPost(path, mediaType, mediaType, representation);
-        return parseResponseWithId(representation, response, CREATED.getStatusCode());
+        Response response = null;
+        try {
+            response = httpPost(path, mediaType, mediaType, representation);
+            return parseResponseWithId(representation, response, CREATED.getStatusCode());
+        } catch (SDKException e) {
+            log.error(" request : POSTT {} for tenant: {} user: {}", path, platformParameters.getTenantId(), platformParameters.getUser());
```

Comment:
```suggestion
            log.error(" request : POST {} for tenant: {} user: {}", path, platformParameters.getTenantId(), platformParameters.getUser());
```

Reply:
Good catch. Sorry for not seeing it myself

Reply:
@hanneshofmann: Thanks for your good feedback. I finally reduced the error handling to missing information.
So method and url is already reported by another class.
`Error invoking GET http://cumulocity:8111/application/currentApplication
	at com.cumulocity.microservice.subscription.repository.impl.CurrentMicroserviceRepository.handleException`